src/styles/calendar.less

```diff
@@ -1,163 +1,163 @@
 // -------- CALENDAR COMPONENT, MODAL STYLED AT BOTTOM --------
 
 .calendar-event-button.ant-btn {
-  background-color: #fcdfaa !important;
-  position: absolute !important;
+  background-color: #fcdfaa;
+  position: absolute;
   right: 5%;
   top: 7.8%;
   width: fit-content;
-  padding: 0 5% !important;
-  color: #f27712 !important;
-  font-weight: bold !important;
+  padding: 0 5%;
+  color: #f27712;
+  font-weight: bold;
 }
 
 .ant-picker-calendar-header {
-  justify-content: flex-start !important;
-  margin: 0 2% !important;
+  justify-content: flex-start;
+  margin: 0 2%;
   div {
-    background-color: #f1c461 !important;
-    color: #f27712 !important;
+    background-color: #f1c461;
+    color: #f27712;
   }
 }
 
 .ant-radio-button-wrapper {
-  background-color: #fcdfaa !important;
-  color: #f27712 !important;
+  background-color: #fcdfaa;
+  color: #f27712;
 }
 .ant-radio-button-wrapper-checked {
-  background-color: #f1c461 !important;
-  color: white !important;
+  background-color: #f1c461;
+  color: white;
 }
 
 thead {
   background-color: #fcdfaa;
 }
 
 th {
-  padding: 2rem !important;
+  padding: 2rem;
 }
 
 thead tr th {
-  color: #f27712 !important;
+  color: #f27712;
 }
 
 ul {
-  background-color: transparent !important;
+  background-color: transparent;
 }
 
 .ant-badge-not-a-wrapper {
   text-align: center;
-  margin-top: 5% !important;
-  padding: 4px 16px !important;
-  width: 100% !important;
+  margin-top: 5%;
+  padding: 4px 16px;
+  width: 100%;
   .ant-badge-status-text {
-    color: black !important;
-    margin-left: 0 !important;
+    color: black;
+    margin-left: 0;
```

Comment:
Is it necessary to explicitly say that a margin is 0 ? Is it overriding something?